src/Samples/GtkSource-5/SourceView/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Gtk;
+using GtkSource;
+
+GtkSource.Module.Initialize();
+var application = Application.New("org.gir.core", Gio.ApplicationFlags.FlagsNone);
+application.OnActivate += (sender, args) =>
+{
+    var window = ApplicationWindow.New((Application) sender);
+    window.Title = "GtkSource5 Demo";
+    window.SetDefaultSize(300, 300);
+    var buf = GtkSource.Buffer.New(null);
+    var view = GtkSource.View.NewWithBuffer(buf);
+    view.Monospace = true;
+    view.ShowLineNumbers = true;
+    Settings? settings = Settings.GetDefault();
+    if ( (settings?.GtkApplicationPreferDarkTheme ?? false) ||
```

Comment:
From my point of view the condition can be a bit simplified. If you aggree you can change it to:

```CS
    if(settings?.GtkApplicationPreferDarkTheme == true || 
       settings?.GtkThemeName?.ToLower().Contains("dark") == true)
        buf.SetStyleScheme(GtkSource.StyleSchemeManager.GetDefault().GetScheme("Adwaita-dark"));
```

Reply:
Yeah, I'm not sure if it's even worth having this extra code at all, or whether it would be better to keep things as minimal/clear as possible in the example. Without loading this style scheme though, the widget will be white background on gnome with dark mode enabled, so I've left it in for now.